### PR TITLE
Refactor css

### PR DESCRIPTION
**Pull request by amrutac**
CoreOS theme
screen shot 2016-12-01 at 12 45 55 pm
screen shot 2016-12-01 at 12 46 27 pm

Tectonic theme
screen shot 2016-12-01 at 12 44 18 pm
screen shot 2016-12-01 at 12 44 59 pm

### CODE REVIEW

**Comment by ericchiang** [comment]
lgtm

File: web/static/img/email-icon.svg [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 21.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns:sketch="http://www.bohemiancoding.com/sketch/ns"
+	 xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 43 43"
+	 style="enable-background:new 0 0 43 43;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:#84B6F0;}
+	.st1{fill:#FFFFFF;}
+</style>
```
Comment (sym3tri) [inline web/static/img/email-icon.svg]: Can you remove the embedded css from the svg and just add the fill attributes directly?

It's unlikely, but possible the classes could conflict with others in the main page.

File: web/static/img/email-icon.svg [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+<title>Rectangle 290 + Rectangle 291 + Path 232</title>
+<desc>Created with Sketch.</desc>
```
Comment (sym3tri) [inline web/static/img/email-icon.svg]: Also dont need any of this metadata

File: web/static/main.css [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+}
+
+.dex-btn:active {
+  box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.125);
+  outline: none;
+}
+
+.dex-btn-provider {
+  background-color: #fff;
+  color: #333;
```
Comment (sym3tri) [inline web/static/main.css]: should all colors go into the themes?

**Comment by sym3tri** [comment]
Also should use an SVG for the tectonic logo.

**Pull request merged**